index.html

```diff
@@ -0,0 +1,402 @@
+            <div class="section2__border section2_border">
+              <svg
+                class="section2__svg"
+                width="43"
+                height="63"
+                viewBox="0 0 43 63"
+                xmlns="http://www.w3.org/2000/svg"
+              >
+                <path
+                  fill-rule="evenodd"
+                  clip-rule="evenodd"
+                  d="M43 53.5C43 58.74 38.75 63 33.5 63H9.5C4.25 63 0 58.74 0 53.5V9.5C0 4.25 4.25 0 9.5 0H33.5C38.75 0 43 4.25 43 9.5V53.5ZM40 9.5C40 5.91 37.09 3 33.5 3H9.5C5.91 3 3 5.91 3 9.5V53.5C3 57.09 5.91 60 9.5 60H33.5C37.09 60 40 57.09 40 53.5V9.5Z"
+                />
+                <path
+                  fill-rule="evenodd"
+                  clip-rule="evenodd"
+                  d="M43 52.4999C43 53.3199 42.33 53.9999 41.5 53.9999H1.5C0.67 53.9999 0 53.3199 0 52.4999V9.5C0 8.67 0.67 8 1.5 8H41.5C42.33 8 43 8.67 43 9.5V52.4999ZM3 50.9999H40V11H3V50.9999Z"
+                />
+                <path
+                  d="M23.9399 5.70996C23.9399 6.80996 23.0499 7.70996 21.9399 7.70996C20.8399 7.70996 19.9399 6.80996 19.9399 5.70996C19.9399 4.59996 20.8399 3.70996 21.9399 3.70996C23.0499 3.70996 23.9399 4.59996 23.9399 5.70996Z"
+                />
+                <path
+                  d="M24.5698 58H19.5698C18.7498 58 18.0698 57.32 18.0698 56.4999C18.0698 55.6699 18.7498 54.9999 19.5698 54.9999H24.5698C25.3998 54.9999 26.0698 55.6699 26.0698 56.4999C26.0698 57.32 25.3998 58 24.5698 58Z"
+                />
+                <path
+                  d="M24.5 39C23.67 39 23 38.11 23 37.28C23.01 29.75 18.44 28.98 17.92 28.92C17.1 28.84 16.49 28.06 16.57 27.25C16.64 26.43 17.35 25.78 18.17 25.85C18.25 25.86 26.0101 26.78 26.0001 37.3C26.0001 38.13 25.33 39 24.5 39Z"
+                />
+                <path
+                  d="M31.36 38.0002C30.53 38.0002 29.86 37.2702 29.86 36.4402C29.88 23.1401 20.99 22.2701 20.9 22.2601C20.07 22.2001 19.45 21.4701 19.52 20.6401C19.58 19.81 20.32 19.16 21.13 19.25C21.24 19.26 32.88 20.4001 32.86 36.4502C32.86 37.2702 32.19 38.0002 31.36 38.0002Z"
+                />
+              </svg>
+            </div>
+            <h3 class="section2__h3 section2_h3">ANIMATIONS</h3>
+            <p class="section2__text section2_text">
+              Lorem ipsum dolor sit amet, consectetur adipisicing elit, sed do
+              eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut
+              enim ad minim
+            </p>
+          </div>
+        </div>
+      </section>
+    </div>
+    <section class="latest__work">
+      <div class="latest__body">
+        <div class="latest__header">
+          <h2 class="latest__title">Latest Works</h2>
+        </div>
+        <div class="latest__main">
+          <div class="container">
+            <div class="row gy-4">
+              <div class="col-12 col-md-6 col-lg-4">
+                <img src="img/Thumbnail.png" alt="" />
```

Comment:
По БЕМ краще буде дати клас для **img** 
```suggestion
                <img src="img/Thumbnail.png" alt="" class="latest-works__work" />
```